Mathlib/Algebra/Associated.lean

```diff
@@ -6,6 +6,7 @@ Authors: Johannes Hölzl, Jens Wagemaker
 import Mathlib.Algebra.Divisibility.Basic
 import Mathlib.Algebra.GroupPower.Lemmas
 import Mathlib.Algebra.Parity
+import Mathlib.Algebra.Group.Prod
```

Comment:
This is a pretty low file in the hierarchy; I'm not sure if we want to add more imports to it. (See also #7200)

Reply:
Well noted. I have moved the result to `Mathlib.Algebra.BigOperators.Associated`.